### ISSUE TITLE
If the first word at WORK_IN_PROGRESS_PREFIXES starts with <, the hint is shown incorrectly

### ISSUE THREAD

**Issue by InExtremaRes** [go-gitea/gitea#4705]
- Gitea version (or commit ref): 03e558c
(the rest are not relevant or difficult)

## Description

Since #4529 a commit can be marked as _WIP_. The _WIP_ prefix can be configured. We have marked our PR with a `<WIP>` prefix format since a while, so we put that as the default. I have this in my app.ini:

```ini
[repository.pull-request]
; List of prefixes used in Pull Request title to mark them as Work In Progress
WORK_IN_PROGRESS_PREFIXES=<WIP>,WIP:,[WIP]
```

It works pretty well, except for the hint text that doesn't show the prefix to use:

![image](https://user-images.githubusercontent.com/1635491/44107865-0d4a7304-9fcf-11e8-9339-3f1c69296ff5.png)

You can see the hint only says `Start the title with`, without the `<WIP>`. For reference, here is how is showed with default values:

![image](https://user-images.githubusercontent.com/1635491/44108005-732a1864-9fcf-11e8-9ec6-87b51a601d8d.png)

After a little of experimentation, it seems the problem occurs any time the prefix start with a `<` character.

@JulienTant

**Comment by JulienTant** [comment]
Ok i'm going to take a look at this, this probably needs some html escaping before printing. Thanks for reporting

**Comment by JulienTant** [comment]
ICYMI @InExtremaRes  I created a PR to fix the bug :  #4709

**Comment by InExtremaRes** [comment]
Yes. Thank you so much. I'm following the discussion there.
